src/content/getting started/prepare-your-provider-account/aws/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Prepare your provider account for AWS
+linkTitle: Prepare your AWS account
+description: The actions need to be done to prepare your AWS account for starting to build your cloud-native developer platform.
```

Comment:
```suggestion
title: Prepare your AWS account
description: Prepare your AWS account to start building your cloud-native developer platform with Giant Swarm.
```

[repeated for every provider]

I had a better sentence yesterday but lost the comment draft – feel free to improve any of the suggestions.